cdap-common/src/main/java/co/cask/cdap/common/discovery/ResolvingDiscoverable.java

```diff
@@ -59,4 +59,9 @@ private InetSocketAddress resolve(InetSocketAddress bindAddress) {
     }
     return bindAddress;
   }
+
+  @Override
+  public String toString() {
+    return "ResolvingDiscoverable(" + String.valueOf(discoverable) + ")";
```

Comment:
OK, so then I will need a static toString() method that uses the name and address. I guess that would belong into Twill. For the time being, will add one here and use it until we have that in Twill.